src/components/Payroll/PayrollLanding/PayrollLanding.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { RunPayroll } from '../RunPayrollFlow/RunPayroll'
+import { PayrollHistory } from '../PayrollHistory/PayrollHistory'
+import { PayrollConfiguration } from '../PayrollConfiguration/PayrollConfiguration'
+import { PayrollEditEmployee } from '../PayrollEditEmployee/PayrollEditEmployee'
+import { PayrollList } from '../PayrollList/PayrollList'
+import { PayrollOverview } from '../PayrollOverview/PayrollOverview'
+import type { BaseComponentInterface } from '@/components/Base/Base'
+import { BaseComponent } from '@/components/Base/Base'
+import { useI18n } from '@/i18n'
+import { useComponentContext } from '@/contexts/ComponentAdapter/useComponentContext'
+
+interface PayrollLandingProps extends BaseComponentInterface<'Payroll.PayrollLanding'> {
+  companyId: string
+}
+
+export function PayrollLanding(props: PayrollLandingProps) {
+  return (
+    <BaseComponent {...props}>
+      <Root {...props} />
+    </BaseComponent>
+  )
+}
+
+export const Root = ({ onEvent, companyId }: PayrollLandingProps) => {
+  const [selectedTab, setSelectedTab] = useState('run-payroll')
+  const { Tabs } = useComponentContext()
+
+  useI18n('Payroll.PayrollLanding')
+  const { t } = useTranslation('Payroll.PayrollLanding')
+
+  const tabs = [
+    {
+      id: 'run-payroll',
+      label: t('tabs.runPayroll'),
+      content: (
+        <RunPayroll
+          companyId={companyId}
+          Configuration={PayrollConfiguration}
+          List={PayrollList}
+          Overview={PayrollOverview}
+          EditEmployee={PayrollEditEmployee}
+          onEvent={onEvent}
+        />
```

Comment:
This is going to cause an issue where the entire payroll experience is in a single tab. We actually just want to render `PayrollList` for the content here.

Then, we would update the RunPayroll component to use this `PayrollLanding` component instead. This will emit events to RunPayroll which can manage transitions between the blocks